tslint.json

```diff
@@ -68,6 +68,6 @@
     "rulesDirectory": [],
     "linterOptions": {
       // TODO: why is tslint looking at .js files anyway?
-      "exclude": ["node_modules/", "**/*.js", "dev-tools/global*"]
+      "exclude": ["node_modules/", "**/*.js", "**/globals*.ts"]
```

Comment:
```suggestion
      "exclude": ["node_modules/", "**/*.js", "*/globals.ts"]
```

seems equivalent?

Reply:
`globals.d.ts`?

Reply:
Can probably remove the `**` though

Reply:
I'm confused that `globals*.ts` would match `global.d.ts` – the latter is missing the first `s`.

Reply:
Also: is `**/*.js` necessary? Is `*.js` enough?

Reply:
`**/*.js` is necessary, and you're right, I had meant `global*.ts` and typoed but clearly it didn't matter. Maybe it skips `.d.ts` files by default (and yet I can't get it to exclude `.js` files? the irony...)?

Updated to be `*/globals.ts`